system-int-test/test/cmr/system_int_test/ingest/provider_ingest_permissions_test.clj

```diff
@@ -240,7 +240,8 @@
                                      {:token provider-admin-update-token
                                       :allow-failure? true})]
         (is (= 401 status))
-        (is (= [(format "Token [%s] has expired" provider-admin-update-token)] errors))))))
+        (is (= [(format "Token [%s] has expired. Note the token value has been partially redacted." 
```

Comment:
Can you change this test to display the actual returned value rather than using the source code to do the comparison?

e.g. `Token [abcdeXXXvwxyz] has expired. Note the token value has been partially redacted.`

Reply:
Done.